android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -827,6 +1057,7 @@ public void createDataChannel(final int peerConnectionId, String label, Readable
             if (-1 != dataChannelId) {
                 dataChannel.registerObserver(new DataChannelObserver(dataChannelId, dataChannel));
                 mDataChannels.put(dataChannelId, dataChannel);
+                registerDataChannelObserver(dataChannelId, dataChannel);
             }
```

Comment:
@zxcpoiu, since I didn't create this PR, I guess I cannot modify it. Could I please ask you to remove the occurrences of `dataChannel.registerObserver(new DataChannelObserver(dataChannelId, dataChannel));` and leave the occurrences of `registerDataChannelObserver(dataChannelId, dataChannel);`? Or do you have a proposal for another way to fix this PR and bring it closer to merging?